### PR TITLE
fix(condo): DOMA-9152 updated unique constraint for BankTransaction

### PR DESCRIPTION
**Pull request by Alllex202**
The field `account` was added to the unique constraint of `BankTransaction` scheme so that it was possible to import transactions that were added between accounts within the same organization

### CODE REVIEW

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/banking/utils/testSchema/index.js [hunk 1]
```diff
@@ -239,7 +239,7 @@ async function updateTestBankIntegrationAccountContext (client, id, extraAttrs =
 async function createTestBankTransaction (client, account, contractorAccount, integrationContext, organization, extraAttrs = {}) {
     if (!client) throw new Error('no client')
     if (!account || !account.id) throw new Error('no account.id')
-    if (!contractorAccount || !contractorAccount.id) throw new Error('no contractorAccount.id')
+    if (contractorAccount && !contractorAccount.id) throw new Error('no contractorAccount.id')
```
Comment (pahaz) [inline apps/condo/domains/banking/utils/testSchema/index.js:242]: what if `contractorAccount` is undefined?

Comment (pahaz) [reply apps/condo/domains/banking/utils/testSchema/index.js:242]: why do we need to create transactions without a contractor?

File: apps/condo/migrations/20240530200526-0396_remove_banktransaction_bank_transaction_unique_organization_importremotesystem_importid_and_more.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create constraint Bank_transaction_unique_organization_account_importRemoteSystem_importId on model banktransaction
```
Comment (pahaz) [inline apps/condo/migrations/20240530200526-0396_remove_banktransaction_bank_transaction_unique_organization_importremotesystem_importid_and_more.js:8]: you need to increase timeout here! it's not so fast to create index!

**Review by toplenboren — COMMENTED** [review]
Approving changes, but please check out @pahaz comments before merging

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/banking/utils/testSchema/index.js (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/banking/utils/testSchema/index.js:242]: `BankTransaction` scheme allows you to leave this `contractorAccount` field empty.

And I managed to get such data from `sbbol`

File: apps/condo/migrations/20240530200526-0396_remove_banktransaction_bank_transaction_unique_organization_importremotesystem_importid_and_more.js (hunk 2 above)
Comment (Alllex202) [reply apps/condo/migrations/20240530200526-0396_remove_banktransaction_bank_transaction_unique_organization_importremotesystem_importid_and_more.js:14]: Fixed

**Review by VKislov — APPROVED** [review]

**Pull request merged**